Applications/Forward/test/subject01_Setup_Forward_Controller.xml

```diff
@@ -60,8 +60,8 @@
 					    indicates the controller will controll all the acuators in the model-->
 					<actuator_list> </actuator_list>
 					<!--Flag (true or false) indicating whether or not the controller is
-					    enabled (ON) should-->
-					<enable_controller> true </enable_controller>
+					    disabled.-->
+					<isDisabled> false </isDisabled>
```

Comment:
@aseth1 Was this change made by hand or by saving the file? How come there's no change in version number? I'd expect the xml tags to be consistent with the XMLDocument version number otherwise something is amiss. Conversions need to be built into the code so that user files continue to function in a predictable manner.

Reply:
ok, thanks for the clarification.